c2corg_api/models/association.py

```diff
@@ -212,6 +212,8 @@ def get_linked_routes(document):
 
     return limit_route_fields(
         DBSession.query(Route).
+        options(joinedload(Route.geometry).load_only(
+            DocumentGeometry.geom_detail)).
```

Comment:
You mean like this https://github.com/c2corg/v6_api/pull/358/commits/cd1ea2f60384c0c56a12d71983a005501ae95277

It seems indeed to work without loading ``geom_detail`` but I don't understand why. The second schema requires this attribute. Isn't it that second schema that is used with got getting the associatied routes of a WP? https://github.com/c2corg/v6_api/blob/associated-wp-geom/c2corg_api/models/association.py#L115-L140